numpy/add_newdocs.py

```diff
@@ -5463,18 +5442,22 @@ def luf(lamdaexpr, *args, **kwargs):
 
     Parameters
     ----------
-    X : array_like
-        First input array.
-    Y : array_like
-        Second input array.
-    out : array_like
-        An array to store the output. Must be the same shape as the
-        output would have.
+    *x : array_like
```

Comment:
Each of the  `*` need escaping like so `\\*` for sphinx.  If you make it a raw string you can use a single `\'.